test/functional/notes_controller_test.rb

```diff
@@ -49,6 +49,85 @@ class NotesControllerTest < ActionController::TestCase
       end
     end
 
+    context "when an Important note is provided" do
+      should "show 'Note recorded' and save a completed note" do
+        post :create,
+             params: {
+               edition_id: @edition.id,
+               note: {
+                 type: "important_note",
+                 comment: @note_text,
+               },
+             }
+
+        @edition.reload
+
+        assert_equal(@note_text, @edition.important_note.comment)
+        assert_redirected_to history_edition_path(@edition)
+        assert_includes flash[:success], "Note recorded"
```

Comment:
Would `assert_equal` be more appropriate here?
```suggestion
        assert_equal "Note recorded", flash[:success]
```